### PR TITLE
🏠[WiP] Organizational Features - Part 2

### PR DESCRIPTION
**Pull request by awtkns**

### CODE REVIEW

File: platform/reworkd_platform/web/api/auth/routes.py
Comment (asim-shrestha) [inline platform/reworkd_platform/web/api/auth/routes.py:1]: Shouldn't this be called views?

File: platform/reworkd_platform/web/api/dependencies.py [hunk 1]
```diff
@@ -18,25 +18,36 @@ def user_crud(
 
 
 async def get_current_user(
+    x_organization_id: Annotated[str | None, Header()] = None,
```
Comment (asim-shrestha) [inline platform/reworkd_platform/web/api/dependencies.py:21]: Why the x_?

**Comment by asim-shrestha** [comment]
mypy failing

File: platform/reworkd_platform/web/api/dependencies.py (hunk 1 above)
Comment (awtkns) [reply platform/reworkd_platform/web/api/dependencies.py:21]: By convention, if a system wants to add its own custom header, it starts with X-. This is so they can be sure their custom headers don't accidentally take the name of any defined header, current or future.

File: next/src/components/sidebar/AuthItem.tsx [hunk 2]
```diff
@@ -31,13 +33,17 @@ const AuthItem: FC<{
...
+        <div className="relative">
+          {user && (
+            <img
+              className="h-9 w-9 rounded-md bg-neutral-800"
+              src={get_avatar(user)}
+              alt="user avatar"
+            />
+          )}
+          {organization && <FaBuilding className="absolute -right-1 -top-1 text-white" />}
+        </div>
```
Comment (asim-shrestha) [inline next/src/components/sidebar/AuthItem.tsx:45]: Will have to update this in the future. I like how linear displays orgs

File: next/src/services/fetch-utils.ts [hunk 3]
```diff
@@ -2,16 +2,24 @@ import type { z } from "zod";
...
+}
+
 export const get = async <T extends z.ZodTypeAny>(
   path: string,
   schema: T,
-  accessToken?: string
+  accessToken?: string,
+  organizationId?: string
 ): Promise<z.infer<T>> => {
+  console.log("get", path, schema, accessToken, organizationId);
```
Comment (asim-shrestha) [inline next/src/services/fetch-utils.ts:19]: remove
```suggestion
```

**Review by asim-shrestha — APPROVED** [review]

**Pull request merged**